letsencrypt/DOCS.md

```diff
@@ -89,6 +89,9 @@ gandi_api_key: ''
 gandi_sharing_id: ''
 transip_username: ''
 transip_api_key: ''
+ionos_endpoint: ''
```

Comment:
```suggestion
```
We should not make this an option. I know that the library makes it an option but I would expect users rarely ever change it. We should hard-code the endpoint in the addon to `https://api.hosting.ionos.com` and not make users look that information up. Unless you have reason to believe users will need to change this a lot, in which case please let me know why that is.